package.json

```diff
@@ -14,7 +14,7 @@
     "test-cypress": "cypress open",
     "test-cypress-build": "cypress run --spec ./cypress/features/build/build*.feature --browser chrome --reporter spec",
     "test-cypress-accessibility": "cypress run --spec ./cypress/features/accessibility/accessibility.feature --browser chrome --reporter spec",
-    "test-cypress-visual": "cypress run --spec './cypress/features/visual/*.feature' --browser chrome --reporter spec",
+    "test-cypress-visual": "cypress run --spec './cypress/features/regression/*.feature' --env TAGS='@applitools',CYPRESS_APPLITOOLS=true --browser chrome --reporter spec",
```

Comment:
if we are running only one `*.feature` file we should pass it in `--spec` parameter, it will speed up those tests

Reply:
isn't it already doing that? `--spec './cypress/features/visual/*.feature'` ??

Reply:
nope, because the file is in the `cypress/features/regression/button.feature` instead of `'./cypress/features/visual/`
Visual folder doesn't exist

Reply:
Sorry miss types my last comment, the command is running all feature files in the 
features/regression directory with the tag applitools

Reply:
` "test-cypress-visual": "cypress run --spec './cypress/features/regression/*.feature' --env TAGS='@applitools',CYPRESS_APPLITOOLS=true --browser chrome --reporter spec",`

Reply:
yes, but cypress will scan all feature files inside

Reply:
OK have removed until i know which files to run